stanford_fellowship.ds.inc

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * @file
+ * stanford_fellowship.ds.inc
+ */
+
+/**
+ * Implements hook_ds_layout_settings_info().
+ */
+function stanford_fellowship_ds_layout_settings_info() {
+  $export = array();
+
+  $ds_layout = new stdClass();
+  $ds_layout->api_version = 1;
+  $ds_layout->id = 'node|stanford_fellowship|full';
```

Comment:
Full content can be disabled - I am using default layout now - makes it less extra code